docs/experiments-disk12.md

```diff
@@ -1,74 +1,60 @@
-(e.g., for query expansion), add the `-storeDocvectors` option.
+For additional details, see explanation of [common indexing options](common-indexing-options.md).
 
-_Hint:_ You can use the `DumpIndex` utility to print out the statistics of the index. Please refer to 
-[DumpIndex References](dumpindex-reference.md) for the statistics of the index
+## Retrieval
 
+Topics and qrels are stored in `src/main/resources/topics-and-qrels/`.
+After indexing has completed, you should be able to perform retrieval as follows:
 
-**Search**:
-
-After indexing is done, you should be able to perform a retrieval run:
-
-```
-sh target/appassembler/bin/SearchCollection -topicreader Trec -index lucene-index.disk12.pos -bm25 \
-  -topics src/main/resources/topics-and-qrels/topics.51-100.txt -output run.disk12.51-100.bm25.txt
 ```
+nohup sh target/appassembler/bin/SearchCollection -topicreader Trec -index lucene-index.disk12.pos+docvectors+rawdocs -topics src/main/resources/topics-and-qrels/topics.51-100.txt  -output run.disk12.51-100.bm25.txt  -bm25 &
+nohup sh target/appassembler/bin/SearchCollection -topicreader Trec -index lucene-index.disk12.pos+docvectors+rawdocs -topics src/main/resources/topics-and-qrels/topics.101-150.txt -output run.disk12.101-150.bm25.txt -bm25 &
+nohup sh target/appassembler/bin/SearchCollection -topicreader Trec -index lucene-index.disk12.pos+docvectors+rawdocs -topics src/main/resources/topics-and-qrels/topics.151-200.txt -output run.disk12.151-200.bm25.txt -bm25 &
 
-**Evaluate**:
-
-Evaluation can be done using `trec_eval`:
-```
-eval/trec_eval.9.0/trec_eval src/main/resources/topics-and-qrels/qrels.51-100.txt run.disk12.51-100.bm25.txt
+nohup sh target/appassembler/bin/SearchCollection -topicreader Trec -index lucene-index.disk12.pos+docvectors+rawdocs -topics src/main/resources/topics-and-qrels/topics.51-100.txt  -output run.disk12.51-100.ql.txt  -ql &
+nohup sh target/appassembler/bin/SearchCollection -topicreader Trec -index lucene-index.disk12.pos+docvectors+rawdocs -topics src/main/resources/topics-and-qrels/topics.101-150.txt -output run.disk12.101-150.ql.txt -ql &
+nohup sh target/appassembler/bin/SearchCollection -topicreader Trec -index lucene-index.disk12.pos+docvectors+rawdocs -topics src/main/resources/topics-and-qrels/topics.151-200.txt -output run.disk12.151-200.ql.txt -ql &
 ```
 
-**Effectiveness Reference**:
-
-##### no stopwords (default)
-
-MAP                                   | BM25   | QL     
---------------------------------------|--------|--------
-TREC-1 Ad Hoc Track: Topics 51-100    | 0.2224 | 0.2164 
-TREC-2 Ad Hoc Track: Topics 101-150   | 0.2008 | 0.2018 
-TREC-3 Ad Hoc Track: Topics 151-200   | 0.2569 | 0.2516 
-**Mean**.                             | **0.2267** | **0.2232** 
+Evaluation can be performed using `trec_eval`:
 
+```
+eval/trec_eval.9.0/trec_eval src/main/resources/topics-and-qrels/qrels.51-100.txt  run.disk12.51-100.bm25.txt       | egrep "^(map|P_30)"
```

Comment:
`trec_eval -m map -m P.30` is more native?

Reply:
Neat, didn't know those options!